healpix/healpy.py

```diff
@@ -16,11 +16,19 @@
 
 RAD2DEG = 180 / np.pi
 
-__all__ = ['nside2resol', 'nside2pixarea', 'nside2npix', 'npix2nside',
-           'pix2ang', 'ang2pix', 'order2nside']
+__all__ = [
+    'nside2resol',
+    'nside2pixarea',
+    'nside2npix',
+    'npix2nside',
+    'pix2ang',
+    'ang2pix',
+    'order2nside',
+]
 
 
 def nside2resol(nside, arcmin=False):
+    """Drop-in replacement for healpy `~healpy.pixelfunc.nside2resol`."""
```

Comment:
Good idea! Does intersphinx work?

Reply:
Yes, I checked locally. Intersphinx to healpy works.